### DISCUSSION TITLE
Newegg de çalışıyor mu ?

### DISCUSSION THREAD

**Discussion by armagan153** [Prince25/StockAlertBot#50] [Q&A] [answered]
Hello,
The bot runs on Amazon, thank you. But when he finds stock, he gets stuck and doesn't look for it.
It works once on Newegg and doesn't search again. Can it be fixed?

**Comment by armagan153** [comment]
![image](https://user-images.githubusercontent.com/77968186/110615465-83282d00-81a4-11eb-8e68-cc3e1217e782.png)
No waiting for newegg connections, could that be the problem?

**Comment by Prince25** [comment, marked as answer]
Hi! Could your issue be the same as https://github.com/PrinceS25/StockAlertBot/discussions/48? Please check and let me know.

**Reply by armagan77** [reply]
Hi, I guess it looks like the same error. No data is displayed, other error messages are not displayed. I'll try it tomorrow.
I wonder if there is any risk of being banned on Newegg. The delay setting is not the same as on Amazon.
By the way thank you very much, a successful bot

**Reply by armagan77** [reply]
And can 2 or 3 bots be run at the same time

**Reply by Prince25** [reply]
Yes, you can run as many instances of the bot as your computer can handle. There's a possibility of any website banning your IP if you send too many requests in a short matter of time. If you're not using proxies, I suggest you either limit the number of URLs or run multiple instances with 5-10 URLs each.

**Reply by armagan153** [reply]
I researched the examples. I have 15 connections, it will wait 30 seconds. The link I added last is in stock. Successful looseness and incompatibility notifications came on first run. However, 15 minutes passed and I did not receive any other notifications. There is stock after 30 seconds, shouldn't I be informed again?
![image](https://user-images.githubusercontent.com/77968186/110805718-6833e680-8292-11eb-8f9b-b154d8c6d94d.png)

**Reply by Prince25** [reply]
As you can see in the line below, you should receive in stock notifications 5 minutes after the last one.
https://github.com/PrinceS25/StockAlertBot/blob/13abdd256df876171eebd5ba2b19c8095bec62e0/stores/newegg.js#L106

I'll run some tests and ensure it functions as expected.

**Reply by Prince25** [reply]
Just to clarify, only the terminal text notifications don't have the 5 minute cooldown so you should receive those every 30 seconds. The fact you haven't seen any more notifications is tells me your program is hanging for some reason. Not having any errors makes it weird. Can you try running only one instance with just 1 url of an item that's in stock?

![Screenshot 2021-03-11 170404](https://user-images.githubusercontent.com/28243638/110881346-10e34580-8295-11eb-85a5-93042955f438.png)

**Reply by Prince25** [reply]
That being said, I fixed a rather major bug. I don't think it'll make a difference for you but download the latest update.
